Updates directions for tutorial to correctly load datasets.

palecur opened the pull request.

tbragin commented:
1. In the "Create Index Patterns" and "Visualize" sections, we need to change "ba*" index pattern to "accounts". Also is there any reason we're using "shakes*" instead of "shakespeare"? What's the point of the wildcard there?

2. I get a mapping conflict on "speech_number" in the "shakespeare" index. Not sure if that is expected?

3. Does it bear mentioning that the curl commands will only work with an unprotected cluster? I think that is our assumption, since this is a Getting Started tutorial, but it might be good to be explicit.  If Shield is in place we'd need something like: curl -XPOST -k -u <username> 'https://localhost:9200/accounts/account/_bulk?pretty' --data-binary @accounts.json. We have that problem across our doc set, I know :)

palecur commented:
Originally I wanted to just illustrate that you *could* wildcard, but the multiple logstash indices make that point better anyhow, so I've reworked the text and updated the index names.

I've also added a note about Shield.

tbragin commented:
LGTM!
